back/src/__tests__/resources/get.test.ts

```diff
@@ -212,4 +212,11 @@ describe('Testing resources GET endpoint', () => {
       expect(resource.voteCount.userVote).toBe(expectedVote)
     })
   })
+  it('should display whether a resource is favorited by a looged user', async () => {
```

Comment:
He añadido un test que crea una entrada en la tabla favorites con el recurso "test-resource-1-blog" y luego comprueba que el GET devuelve los recursos con la propiedad isFavorite y que para el primer recurso da true. 

Ahora el controlador sólo hace una query extra, que es para obtener todos los favorites del user.

Estoy mirando de optimizarlo para que sea a través del include que se añada la propiedad y sin necesidad de ningún query extra.

Reply:
at transformResourcetoApi.ts, added optional ? on isFavorite property, because in some resource controllers TS was complaining about the absence of this property.